client/lib/analytics/ad-tracking/woo/test/is-wooexpress-upgrade.ts

```diff
@@ -0,0 +1,21 @@
+import { type ResponseCartProduct, getEmptyResponseCartProduct } from '@automattic/shopping-cart';
+import { isValidWooExpressUpsell } from '../is-wooexpress-upgrade';
+
+describe( 'isValidWooExpressUpsell', () => {
+	test( 'should return true if the product is a Woo Express plan', () => {
+		const product: ResponseCartProduct = {
+			...getEmptyResponseCartProduct(),
+			product_slug: 'wooexpress-medium-bundle-yearly',
```

Comment:
Can we use `PLAN_WOOEXPRESS_MEDIUM` directly instead of the hard-coded slug?
```suggestion
			product_slug: PLAN_WOOEXPRESS_MEDIUM,
```